src/app/main/components/recipes-list/components/recipe-card/recipe-card.component.scss

```diff
@@ -1,15 +1,33 @@
 @import '/src/styles/base/variables.scss';
```

Comment:
you can use the settings in angular.json 
```
"stylePreprocessorOptions": {
              "includePaths": [
                "/src/styles/base/"
              ]
 },
```

after that you will be able to use such import of the variables file 
`@import 'variables'`

don't forget change the import on all of you're files